synapse/storage/schema/delta/32/reports.sql

```diff
@@ -15,6 +15,8 @@
 
 
 CREATE TABLE event_reports(
+    id BIGINT NOT NULL,
+    received_ts BIGINT NOT NULL,
```

Comment:
I really don't see the benefit of adding an index we don't use? I don't even know how this data is going to be used.